lib/infusionsoft/api.rb

```diff
@@ -221,5 +221,11 @@ def fetch_contacts_with_tag(tag_id, custom_fields = [], page = 0)
       end
       contacts
     end
+
+    def get_contacts(page = 0, custom_fields = [], data={})
```

Comment:
I have two questions on this:
1. Why is it not named "fetch_contacts"?
2. Why is argument order different from fetch_contacts_with_tag? (custom_fields, page)